ipatests/test_integration/test_topology.py

```diff
@@ -10,16 +10,15 @@
 from ipatests.test_integration.base import IntegrationTest
 from ipatests.pytest_ipa.integration import tasks
 from ipatests.pytest_ipa.integration.env_config import get_global_config
-from ipalib.constants import DOMAIN_SUFFIX_NAME
 from ipatests.util import assert_deepequal
 
 config = get_global_config()
 reasoning = "Topology plugin disabled due to domain level 0"
 
 
-def find_segment(master, replica):
+def find_segment(master, replica, suffix):
```

Comment:
When an existing function signature is modified, you should provide a default value for the new parameter:
`def find_segment(master, replica, suffix=DOMAIN_SUFFIX_NAME):`

This way the existing code already using the function doesn't have to be modified (you can remove the change on line 138).